### DISCUSSION TITLE
利用Alist挂载阿里云盘后群晖Cloud sync上传文件失败

### DISCUSSION THREAD

**Discussion by lc881125** [AlistGo/alist#5483] [Q&A]
- Alist成功设置挂载，cloud sync设置成功，从阿里云盘下载文件没有问题。
- 通过cloud sync上传文件是出错，具体表现：1G以内的文件随机成功上传，4G以上内容上传失败，上传进度达到100%后仍然有传输速度，过几分钟后会重新上传，如此循环。且上传失败后不会提报信息(Cloud sync中没有事件记录)
![image](https://github.com/alist-org/alist/assets/44010188/48f8cece-7e03-46f0-823b-baf1224f3f3d)

- Alist版本号：3.28.0 
- Alist设置
    - 缓存过期时间：180分钟
    - 代理：本地代理
    - Oauther：https://api.xhofe.top/alist/ali_open/token

另：用file Station挂载云盘，拖拽文件上传有相同问题。上传失败报错：抱歉，此用户帐户权限不足，无法执行此操作。

**Comment by lqzhgood** [comment]
参考 https://github.com/alist-org/alist/issues/5403

最后我提了一个暂时手动解决 CloudSync 上传大文件的办法

**Comment by daxiaraoming** [comment]
我用CloudSync大文件也是一直失败重试,没有错误日志,而且大量小文件很慢,后来发现Hyper Backup本身就支持webdav,速度给力,直接不用CloudSync了

**Reply by zhuiyue132** [reply]
Hyper Backup 你是用的 alist 的 webdav 吗。我一直备份失败。

**Reply by daxiaraoming** [reply]
我没用alist
